src/footprintmonitoring/azext_footprintmonitoring/generated/_params.py

```diff
@@ -0,0 +1,198 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('footprintmonitoring profile list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('footprintmonitoring profile show') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('profile_name', options_list=['--name', '-n'], type=str, help='Name of the Footprint profile '
+                   'resource.', id_part='name')
+
+    with self.argument_context('footprintmonitoring profile create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('profile_name', options_list=['--name', '-n'], type=str, help='Name of the Footprint profile '
+                   'resource.')
+        c.argument('tags', tags_type)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx),
+                   validator=get_default_location_from_resource_group)
+        c.argument('provisioning_state',
```

Comment:
Should this one be exposed to customer?

Reply:
@jonunezd  if it can't be updated, you can add a `"readOnly": true` in the swagger and regenerate it.

Reply:
I regenerated locally, will send a separate PR under the specs repo with this change shortly.

Reply:
PR at https://github.com/Azure/azure-rest-api-specs-pr/pull/1792